applications/serial_lte_modem/src/gpio/slm_at_gpio.c

```diff
@@ -0,0 +1,312 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <logging/log.h>
+#include <zephyr.h>
+#include <stdio.h>
+#include "slm_util.h"
+#include "slm_at_gpio.h"
+
+LOG_MODULE_REGISTER(slm_gpio, CONFIG_SLM_LOG_LEVEL);
+
+/* global functions defined in different resources */
+void rsp_send(const uint8_t *str, size_t len);
```

Comment:
Please include "slm_at_host.h" instead, using the APIs defined there

Reply:
include slm_at_host.h